template/{{.input_root_dir}}/README.md.tmpl

```diff
@@ -4,17 +4,145 @@ This directory contains an ML project based on the default
+
+| Component                  | Description                                                                                                                                                                                                                                                                                                                                             |
+|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ML Code                    | Example ML project code, with unit tested Python modules and notebooks                                                                                                                                                                                                                                                                                  |
+| ML Resource Config as Code | ML pipeline resource config (training and batch inference job schedules, etc) configured and deployed through [databricks CLI bundles]({{ template `generate_doc_link` (map (pair "cloud" .input_cloud) (pair "path" "dev-tools/cli/bundle-cli.html")) }})                                                                                              |
+| CI/CD                      | {{ if (eq .input_cicd_platform `github_actions`) }}[GitHub Actions](https://github.com/actions) workflows to test and deploy ML code and resources {{ else if (eq .input_cicd_platform `azure_devops`) }}[Azure DevOps Pipelines](https://azure.microsoft.com/en-gb/products/devops/pipelines/) to test and deploy ML code and resources{{ end }}      |
+
+contained in the following files:
+
+```
+{{ template `root_dir` .}}        <- Root directory. Both monorepo and polyrepo are supported.
+│
+├── {{template `project_name_alphanumeric_underscore` .}}       <- Contains python code, notebooks and ML resources related to one ML project. 
+│   │
+│   ├── requirements.txt        <- Specifies Python dependencies for ML code (for example: model training, batch inference).
+│   │
+│   ├── databricks.yml          <- databricks.yml is the root ML resource config file for the ML project that can be loaded by databricks CLI bundles. It defines the bundle name, workspace URL and resource config component to be included.
+│   │
+{{ if and (eq .input_include_feature_store `no`) (eq .input_include_mlflow_recipes `no`) -}}
+│   ├── training                <- Training folder contains Notebook that trains and registers the model.
+│   │
+│   ├── validation              <- Optional model validation step before deploying a model.
+│   │
+│   ├── monitoring              <- Model monitoring, feature monitoring, etc.
```

Comment:
Just call this Lakehouse monitoring?

Reply:
this could be a customized monitoring module users created too. Since we have not build the Lakehouse monitoring integration to Stacks yet. I think it makes more sense to keep it as "monitoring"

Reply:
Agreed, makes sense. I was referring more to the "Model monitoring, feature monitoring, etc." line, but we don't need to change it.